packages/host/app/helpers/truth-helpers.ts

```diff
@@ -7,3 +7,6 @@ export function gt<T>(a: T, b: T): boolean {
 export function not(a: any): boolean {
   return !a;
 }
+export function or(a: boolean, b: boolean): boolean {
```

Comment:
With growing need for basic helpers maybe we can start using https://github.com/jmurphyau/ember-truth-helpers?

Reply:
ours are actually better because they are in TS. but honestly, it's so trivial now to make these with strict mode templates that it feel like extra overhead to import something for these.